creation/web_base/frontend/singularity_script_example.sh

```diff
@@ -54,6 +54,11 @@ warn_raw() {
 [[ -z "$glidein_config" ]] && [[ -e "$GWMS_THIS_SCRIPT_DIR/../glidein_config" ]] &&
     glidein_config="$GWMS_THIS_SCRIPT_DIR/../glidein_config"
 
+# import add_config_line function
+add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | awk '{print $2}')
+# shellcheck source=./add_config_line.source
+. "$add_config_line_source"
+
```

Comment:
remove this part. 
"$glidein_config" could not exist
"$add_config_line_source" could be empty

Reply:
done